### DISCUSSION TITLE
Connection error

### DISCUSSION THREAD

**Discussion by tsvetkovsecurity** [projectdiscovery/nuclei#3335] [Q&A]
From today i got this:

Could not initialize interactsh client: could not create client: could not register to servers: could not make register request: POST https://oast.me/register giving up after 1 attempts: Post "https://oast.me/register": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by gmustafakose** [comment]
same error???
nuclei engine 2.8.9
nuclei template 9.3.9

**Comment by olearycrew** [comment]
Can you provide the output of:

`nuclei -hc`

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
